dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -25,34 +25,31 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 {
     PushNotificationReceivedEventArgs::PushNotificationReceivedEventArgs(winrt::IBackgroundTaskInstance const& backgroundTask):
         m_backgroundTaskInstance(backgroundTask),
-        m_rawNotification(GetByteArrayFromBuffer(backgroundTask.TriggerDetails().as<RawNotification>().ContentBytes())),
-        m_isBIAvailable(true) {}
+        m_rawNotificationPayload(BuildPayload(backgroundTask.TriggerDetails().as<RawNotification>().ContentBytes().data(), backgroundTask.TriggerDetails().as<RawNotification>().ContentBytes().Length())),
+        m_isUnpackagedApp(true) {}
```

Comment:
Probably at this point, the best to do is to rename this var. All checks throughout the code makes me think if this is in fact `m_isPackagedApp`.